trino-lb/src/scaling/mod.rs

```diff
@@ -153,8 +158,8 @@ impl Scaler {
                     // Cluster groups that don't need scaling are missing from the `scaling_config`.
                 }
 
-                Some(match scaler {
-                    ScalerConfig::Stackable(scaler_config) => {
+                Some(match &scaler.implementation {
```

Comment:
thought: Nothing to do with this PR, but I personally prefer a `let blah`, and then a simple `Some(blah)` returned rather than a complex expression inside the `Some(...)`.

Reply:
Depends on personal preference I guess :)
9ed60d2